common-lib/src/main/scala/com/gu/mediaservice/lib/metadata/ImageMetadataConverter.scala

```diff
@@ -30,22 +30,46 @@ object ImageMetadataConverter {
       dateTaken           = (fileMetadata.exifSub.get("Date/Time Original Composite") flatMap parseRandomDate) orElse
                             (fileMetadata.iptc.get("Date Time Created Composite") flatMap parseRandomDate) orElse
                             (fileMetadata.xmp.get("photoshop:DateCreated") flatMap parseRandomDate),
-      description         = fileMetadata.iptc.get("Caption/Abstract"),
-      credit              = fileMetadata.iptc.get("Credit"),
-      byline              = fileMetadata.iptc.get("By-line"),
-      bylineTitle         = fileMetadata.iptc.get("By-line Title"),
-      title               = fileMetadata.iptc.get("Headline"),
-      copyrightNotice     = fileMetadata.iptc.get("Copyright Notice"),
+      description         = fileMetadata.xmp.get("dc:description[1]") orElse
+                            fileMetadata.iptc.get("Caption/Abstract") orElse
+                            fileMetadata.exif.get("Image Description"),
+      credit              = fileMetadata.xmp.get("photoshop:Credit") orElse
+                            fileMetadata.iptc.get("Credit"),
+      // FIXME: Have a way of dealing with arrays, like [1] here. Not much of a practical problem in our corpus, but still..
+      byline              = fileMetadata.xmp.get("dc:creator[1]") orElse
+                            fileMetadata.iptc.get("By-line") orElse
+                            fileMetadata.exif.get("Artist"),
+      bylineTitle         = fileMetadata.xmp.get("photoshop:AuthorsPosition") orElse
+                            fileMetadata.iptc.get("By-line Title"),
+      title               = fileMetadata.xmp.get("photoshop:Headline") orElse
+                            fileMetadata.iptc.get("Headline"),
+      copyrightNotice     = fileMetadata.xmp.get("dc:Rights") orElse
+                            fileMetadata.iptc.get("Copyright Notice"),
       // FIXME: why default to copyrightNotice again?
-      copyright           = fileMetadata.exif.get("Copyright") orElse fileMetadata.iptc.get("Copyright Notice"),
-      suppliersReference  = fileMetadata.iptc.get("Original Transmission Reference") orElse fileMetadata.iptc.get("Object Name"),
-      source              = fileMetadata.iptc.get("Source"),
-      specialInstructions = fileMetadata.iptc.get("Special Instructions"),
+      // I don't get what you mean, Mate. This makes no sense: our copyright and copyrightNotice fields
+      // read from equivalent fields in a stupid order. Not fixing, cause wrong logic is wide-spread.
```

Comment:
This comment is confusing to me - perhaps be a bit more descriptive than conversational. Something like "Picture providers often will put copyright information in the copyright notice field so we have to default to that in order to be safe"?

Reply:
Here we invented two fields for one field (expressed in different schemas). We should just call it `copyrightNotice` and replace all uses of `copyright` with it. But this was too scary for me.